microsoft-edge/extensions-chromium/developer-guide/api-support.md

```diff
@@ -89,7 +127,7 @@ Microsoft Edge doesn't support the following Extension APIs:
 <!-- ====================================================================== -->
 > [!NOTE]
 > Portions of this page are modifications based on work created and [shared by Google](https://developers.google.com/terms/site-policies) and used according to terms described in the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
-> The original page is found [here](https://developer.chrome.com/apps/external_extensions).
+> The original page is found [here](https://developer.chrome.com/docs/extensions/reference/api).
```

Comment:
5: [here] link - what's the best webpage to link to?  @Bhuvana-Priya

The present article is the API list article, updated for MV3.

Reply:
Conclusion from MH & BP: the updated [here] link is correct.